server/src/config.rs

```diff
@@ -95,6 +107,8 @@ mod tests {
         std::env::set_var("COMPLIANCE_PORT_GRPC", "12354");
         std::env::set_var("STORAGE_HOST_GRPC", "test_host_storage_grpc");
         std::env::set_var("STORAGE_PORT_GRPC", "12345");
+        std::env::set_var("GIS_HOST_GRPC", "test_host_gis_grpc");
+        std::env::set_var("GIS_PORT_GRPC", "12345");
```

Comment:
Maybe use a different test number than storage, just to make sure it's really picking up the right env var.

Reply:
https://github.com/Arrow-air/svc-scheduler/pull/39/commits/b98f89ac17fa9427aacb7e0021392d176fb42f10